release/models/types/openconfig-yang-types.yang

```diff
@@ -117,6 +124,16 @@ module openconfig-yang-types {
       "A string consisting of a hexadecimal characters.";
   }
 
+  typedef hex-string-prefixed {
```

Comment:
I think this can be named better. Not a great suggestion but how about rfc6991-hex-string ?